1-js/11-async/08-async-await/article.md

```diff
@@ -286,22 +284,22 @@ let results = await Promise.all([
 ]);
 ```
 
-In case of an error, it propagates as usual: from the failed promise to `Promise.all`, and then becomes an exception that we can catch using `try..catch` around the call.
+In the case of an error, it propagates as usual, from the failed promise to `Promise.all`, and then becomes an exception that we can catch using `try..catch` around the call.
 
 ````
 
 ## Summary
 
 The `async` keyword before a function has two effects:
 
-1. Makes it always return a promise.
-2. Allows to use `await` in it.
+1. Makes it always return a promise
+2. Allows `await` to be used in it
 
 The `await` keyword before a promise makes JavaScript wait until that promise settles, and then:
 
-1. If it's an error, the exception is generated, same as if `throw error` were called at that very place.
+1. If it's an error, the exception is generated — same as if `throw error` was called at that very place.
```

Comment:
M-dash slightly better, could go either way on were/was in this context.

Reply:
I've retained the "were".